implementations/micrometer-registry-stackdriver/src/main/java/io/micrometer/stackdriver/StackdriverMeterRegistry.java

```diff
@@ -138,11 +139,57 @@ public void start(ThreadFactory threadFactory) {
 
     @Override
     public void stop() {
-        if (client != null)
-            client.shutdownNow();
         super.stop();
     }
```

Comment:
Is it necessary to have an overridden method that just calls its `super()`?
If we remove this method the `stop()` from parent is called anyway.

Reply:
Ah, it looks like removing the override breaks binary compatibility even if it is source compatible, which @dimovelev found out when they tried to remove it and added it back.